understanding/consistent-identification.html

```diff
@@ -58,185 +56,139 @@
+                  				達成基準 3.2.4:</span>一貫した識別性 (レベル AA)
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
-                  <p>Components that have the <a href="#dfn-same-functionality">same functionality</a> within a <a href="#dfn-set-of-web-pages">set of Web pages</a> are identified consistently.
+                  <p><a href="#dfn-set-of-web-pages">ウェブページ一式</a>の中で<a href="#dfn-same-functionality">同じ機能</a>を有するコンポーネントは、一貫して識別できる。
                      
                   </p>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Actions are more predictable across pages.</dd>
+                  <dt>目標</dt>
+                  <dd>ページ間での動作をより予測可能にする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Identify repeating functions consistently.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>繰り返される機能を一貫して識別する。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>Consistently identified actions are especially important to people with disabilities.</dd> 
+                  <dt>なぜそれが重要か</dt>
+                  <dd>一貫して識別される動作は、障害のある人にとって特に重要である。</dd> 
                   
                </dl>
             </section>
             <section id="intent">
-               <h2>Intent</h2>
-               <p>The intent of this Success Criterion is to ensure consistent identification of functional
-                  components that appear repeatedly within a set of Web pages. A strategy that people
-                  who use screen readers use when operating a Web site is to rely heavily on their familiarity
-                  with functions that may appear on different Web pages. If identical functions have
-                  different labels (or, more generally, a different <a href="https://www.w3.org/TR/accname-aam-1.1/">accessible name</a>)
-                  on different Web pages, the site will be considerably more difficult
-                  to use. It may also be confusing and increase the cognitive load for people with cognitive
-                  limitations. Therefore, consistent labeling will help. 
+               <h2>意図</h2>
+               <p>この達成基準の意図は、ウェブページ一式で繰り返して表示される機能的なコンポーネントを一貫して識別できるようにすることである。ウェブサイトを操作する際にスクリーンリーダーを使用している利用者は、複数のウェブページで使われている機能に馴染みがあるかどうかに大きく依存している。全く同じ機能が、複数のウェブページ毎に異なるラベル (すなわち、より一般的には、異なる <a href="https://www.w3.org/TR/accname-aam-1.1/">アクセシブルな名前 (accessible name)</a>) を付けられている場合、そのウェブサイトはかなり使いづらいものになってしまう。それは、認知的限界のある人々に混乱をもたらし、認知的負荷を増大させてしまうことがある。したがって、一貫したラベルが役立つ。
```

Comment:
括弧内の or が「すなわち」となっていますが「もしくは」「又は」ではないでしょうか。この仕様では可視のものをラベル、支援技術に通知されるものを名前と呼んでいて両者は異なる概念なので、この or は普通に or の意味かと。

```suggestion
               <p>この達成基準の意図は、ウェブページ一式で繰り返して表示される機能的なコンポーネントを一貫して識別できるようにすることである。ウェブサイトを操作する際にスクリーンリーダーを使用している利用者は、複数のウェブページで使われている機能に馴染みがあるかどうかに大きく依存している。全く同じ機能が、複数のウェブページ毎に異なるラベル (もしくは、より一般的には、異なる <a href="https://www.w3.org/TR/accname-aam-1.1/">アクセシブルな名前 (accessible name)</a>) を付けられている場合、そのウェブサイトはかなり使いづらいものになってしまう。それは、認知的限界のある人々に混乱をもたらし、認知的負荷を増大させてしまうことがある。したがって、一貫したラベルが役立つ。
```

Reply:
「又は」としました。